docs/pages/access-controls/guides/mfa-for-admin-actions.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: MFA for Administrative Actions
+description: Require MFA checks to perform administrative actions.
+---
+
+Teleport can be configured to require additional multi-factor authentication
+checks to perform administrative actions through `tctl`, `tsh`, the Web UI,
+Teleport Connect, and any other Teleport client.
+
+Examples of administrative actions include, but are not limited to:
+
+- Resetting or recovering user accounts
+- Inviting new users
+- Updating cluster configuration resources
+- Modifying access management resources
+- Approving access requests
+- Generating new join tokens
+- Impersonation
+- Creating new bots for Machine ID
+
+This is an advanced security feature that protects users against compromises of
+their on-disk Teleport certificates.
+
+<Notice type="warning">
+  When MFA for administrative actions is enabled, user certificates produced
+  with `tctl auth sign` will no longer be suitable for automation due to the
+  additional MFA checks.
+
+  We recommend using [Machine ID](../../machine-id/getting-started.mdx) to
+  issue certificates for automated workflows, which uses role impersonation
+  that is not subject to MFA checks.
+
+  Certificates produced with `tctl auth sign` directly on an Auth service
+  instance using the super-administrative are not subject to MFA checks to support
+  legacy self-hosted setups.
```

Comment:
```suggestion
  Certificates produced with `tctl auth sign` directly on an Auth Service
  instance using the super-administrative are not subject to MFA checks to support
  legacy self-hosted setups.
```

Reply:
Is "the super-administrative" intended as a modifier for a missing noun (e.g., "role")?